src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -132,6 +132,12 @@ public void Start(ApplicationData applicationData)
 
             Application.Run();
 
+            // By emitting this, we fix an issue where arrow keys don't work in the console
+            // because .NET requires application mode to support Arrow key escape sequences
+            // Esc[?1h - Set cursor key to application mode
+            // See http://ascii-table.com/ansi-escape-sequences-vt-100.php
+            Console.Write("\u001b[?1h");
```

Comment:
Is `Close()` below used and should this be there? Or in Dispose() so that it gets emitted even if something fails?

Reply:
Good point. Moved it to Dispose.